qiskit_experiments/library/characterization/analysis/drag_analysis.py

```diff
@@ -151,6 +151,7 @@ def _run_curve_fit(
         self,
         curve_data: curve.CurveData,
         models: List[lmfit.Model],
+        init_guesses: Dict[str, float],
```

Comment:
Out of curiosity, what triggered this change?

Reply:
Now we don't need to update drag with polished implementation in https://github.com/Qiskit/qiskit-experiments/pull/824/commits/32e0fcfd81825c6c8d19640652880324b2476155